hw-1/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.epam</groupId>
+    <artifactId>hw-1</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <modules>
+        <module>jmp-bank-api</module>
+        <module>jmp-dto</module>
+        <module>jmp-cloud-bank-impl</module>
+        <module>jmp-cloud-service-impl</module>
+        <module>application</module>
+    </modules>
+    <dependencies>
+        <dependency>
```

Comment:
If you want to use this module somewhere you will need to put it in the dependencyManagement tag. Othrewise, that results in compilation failure.

```sh
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] 'dependencies.dependency.[com.epam:jmp.dto:1.0-SNAPSHOT]' for com.epam:jmp.dto:1.0-SNAPSHOT is referencing itself. @ com.epam:hw-1:1.0-SNAPSHOT, /path/to/project/JMP-2023/hw-1/pom.xml, line 19, column 21
```